stories/demos/AllFeatures.tsx

```diff
@@ -85,11 +85,7 @@ export default function AllFeatures() {
       width: 200,
       resizable: true,
       formatter(props) {
-        return (
-          <div onClick={() => gridRef.current?.openCellEditor(props.rowIdx, props.column.idx)}>
-            {props.row.title}
-          </div>
```

Comment:
This stopped working and I think the following happens
- Click on the cell.
- Event is handled by the formatted first and it calls `openCellEditor` which opens the editor
- Event is handled by the cell, it fires `SELECT_CELL` event which selects the cell and hides the editor.

I think previously there was some timing issue

Reply:
`event.stopPropagation()` might fix that.